packages/boost-libs/build

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```

Comment:
- The metrics module only directly pulls in `system` as `asio` requires it (despite otherwise being header-only).
- The avro library pulls in the other three (`filesystem`/`iostreams`/`program_options`).
The mesos module meanwhile is just using header-only components IIRC